awswrangler/_config.py

```diff
@@ -57,7 +57,7 @@ class _ConfigArg(NamedTuple):
     "redis_password": _ConfigArg(dtype=str, nullable=True),
     "ignore_reinit_error": _ConfigArg(dtype=bool, nullable=True),
     "include_dashboard": _ConfigArg(dtype=bool, nullable=True),
-    "log_to_driver": _ConfigArg(dtype=bool, nullable=True),
+    "log_to_driver": _ConfigArg(dtype=bool, nullable=True, loaded=True, default=False),
```

Comment:
I am not sure what the `default` should be here (True or False), i.e. do we want verbose worker logs to show by default or not?

Reply:
I'd vote we follow Ray's default for now. Nice call putting this in config.